### PR TITLE
chore: 🤖 create relative date helper

### PR DESCRIPTION
**Pull request by dvonanderson**
Create a relative date helper that outputs "time" ago format.

✅ Closes: ICU-5847

:tickets: [Jira ticket](https://hashicorp.atlassian.net/browse/JIRA_TICKET_NUMBER)

## Description
We needed a way to display a relative date in “time” ago format, so instead of building logic for that one instance in `admin/ui` , we are moving the logic from current `<RelativeDatetimeLive/>` component and make it into a helper that can be used in any template throughout the application if needed.

### Screenshots (if appropriate):
Screen Shot 2022-08-25 at 10 25 43
Screen Shot 2022-08-25 at 10 26 20

### CODE REVIEW

File: addons/core/addon/helpers/relative-datetime-live.js [hunk 1]
```diff
@@ -0,0 +1,67 @@
...
+      },
+      unitMappings[0]
+    );
+    const scaledDelta =
+      delta < 0
+        ? Math.ceil(delta / greatestMeaningfulUnit.value)
+        : Math.floor(delta / greatestMeaningfulUnit.value);
+    const unit = greatestMeaningfulUnit.unit;
+    return this.intl.formatRelative(scaledDelta, {
+      unit: unit,
```
Comment (ZedLi) [inline addons/core/addon/helpers/relative-datetime-live.js:63]: This variable could either be inlined or just 
```suggestion
      unit,
```

File: addons/core/tests/integration/helpers/relative-datetime-live-test.js [hunk 2]
```diff
@@ -0,0 +1,38 @@
+import { module, test } from 'qunit';
+import { setupRenderingTest } from 'ember-qunit';
+import { render } from '@ember/test-helpers';
+import { hbs } from 'ember-cli-htmlbars';
+
+module('Integration | Helper | relative-datetime-live', function (hooks) {
+  setupRenderingTest(hooks);
+
+  test('it renders relative date in "time" ago format', async function (assert) {
```
Comment (ZedLi) [inline addons/core/tests/integration/helpers/relative-datetime-live-test.js:9]: Maybe show a few examples of a date in the future as well?

**Review by ZedLi — APPROVED** [review]
LGTM

**Review by randallmorey — APPROVED** [review]
Great work refactoring this into a helper!

File: addons/core/tests/integration/helpers/relative-datetime-live-test.js (hunk 2 above)
Comment (dvonanderson) [reply addons/core/tests/integration/helpers/relative-datetime-live-test.js:9]: @ZedLi meaning?

Comment (ZedLi) [reply addons/core/tests/integration/helpers/relative-datetime-live-test.js:9]: As in some tests with the date in the future to show it works not just with dates in the past

Comment (dvonanderson) [reply addons/core/tests/integration/helpers/relative-datetime-live-test.js:9]: @ZedLi fixed

**Pull request merged**